### PR TITLE
Fixing adding ovc templates

### PR DESCRIPTION
**Pull request by john-kheir**

### CODE REVIEW

File: tests/integration_tests/testsuite/ovc_tests/a_basic/accounts_tests.py [hunk 1]
```diff
@@ -197,3 +198,94 @@ def test004_account_add_delete_user(self):
...
         self.log('%s ENDED' % self._testID)
+
+    def test005_get_account_info(self):
+        """ ZRT-OVC-022
+        *Test case for getting account info*
+
+        **Test Scenario:**
+
+        #. Create an account (A1).
+        #. Get A1 and check its info.
```
Comment (Dinaamagdy) [inline tests/integration_tests/testsuite/ovc_tests/a_basic/accounts_tests.py]: add step which define that you get account info with account_info action

File: tests/integration_tests/testsuite/ovc_tests/a_basic/accounts_tests.py [hunk 2]
```diff
@@ -197,3 +198,94 @@ def test004_account_add_delete_user(self):
...
+        """
+        self.log('%s STARTED' % self._testID)
+
+        self.log('Create an account (A1)')
+        openvcloud_ser_name = self.random_string()
```
Comment (Dinaamagdy) [inline tests/integration_tests/testsuite/ovc_tests/a_basic/accounts_tests.py]: what's ser meaning , I think need more clear name

File: tests/integration_tests/testsuite/ovc_tests/a_basic/accounts_tests.py [hunk 3]
```diff
@@ -197,3 +198,94 @@ def test004_account_add_delete_user(self):
...
+        *Test case for setting vdcuser groups*
+
+        **Test Scenario:**
+
+        #. Create vdc user, should succeed.
+        #. Set user groups and check if it was set.
+        """
+        self.log('%s STARTED' % self._testID)
+
+        self.log(' vdc user, should succeed')
```
Comment (Dinaamagdy) [inline tests/integration_tests/testsuite/ovc_tests/a_basic/accounts_tests.py]: why create new vdc user? why don't use the created user on setup ?

File: tests/integration_tests/testsuite/ovc_tests/a_basic/cloudspaces_tests.py [hunk 4]
```diff
@@ -32,36 +32,45 @@ def setUp(self):
                              }
         self.CLEANUP["accounts"].append(self.acc1)
 
-    @unittest.skip('https://github.com/openvcloud/0-templates/issues/47')
+    def tearDown(self):
```
Comment (Dinaamagdy) [inline tests/integration_tests/testsuite/ovc_tests/a_basic/cloudspaces_tests.py]: we delete account on utils which delete cloudspaces inside it too, so why we add teardown here too ?

File: tests/integration_tests/testsuite/ovc_tests/a_basic/vms_tests.py [hunk 5]
```diff
@@ -20,13 +21,28 @@ def setUp(self):
...
-        self.temp_actions = {'openvcloud': {'actions': ['install']},
+        self.temp_actions = {'sshkey': {'actions': ['install']},
+                             'openvcloud': {'actions': ['install']},
                              'account': {'actions': ['install']},
                              'vdcuser': {'actions': ['install']},
                              'vdc': {'actions': ['install']},
                              'node': {'actions': ['install']}}
         self.CLEANUP["accounts"].append(self.acc1)
 
+    def tearDown(self):
```
Comment (Dinaamagdy) [inline tests/integration_tests/testsuite/ovc_tests/a_basic/vms_tests.py]: same as cloudspace teardown ?

Comment (john-kheir) [reply tests/integration_tests/testsuite/ovc_tests/a_basic/vms_tests.py]: same

File: tests/integration_tests/testsuite/ovc_tests/a_basic/cloudspaces_tests.py (hunk 4 above)
Comment (john-kheir) [reply tests/integration_tests/testsuite/ovc_tests/a_basic/cloudspaces_tests.py]: now you need to clean the vms before you delete the cloudspace and u need to delete vdcs before u delete account.. other than this u will get an error saying "cloudspace or account is not empty"

File: tests/integration_tests/testsuite/ovc_tests/a_basic/accounts_tests.py (hunk 2 above)
Comment (john-kheir) [reply tests/integration_tests/testsuite/ovc_tests/a_basic/accounts_tests.py]: means service .. i changed to be more clear

File: tests/integration_tests/testsuite/ovc_tests/a_basic/accounts_tests.py (hunk 3 above)
Comment (john-kheir) [reply tests/integration_tests/testsuite/ovc_tests/a_basic/accounts_tests.py]: Done.. because simply it is chance to verify dsl for creating user

**Pull request merged**